### ISSUE TITLE
расчет НДС на товары по прейскуранту

### ISSUE THREAD

**Issue by gsbelarus** [GoldenSoftwareLtd/gedemin#280]
Originally reported on Google Code with ID 280

```
есть ссылка на документы и пример расчета, как должен браться НДС при
покупке-продаже товара по прейскуранту фиксированных розничных цен.
Глав Бух 2006 №23(71)(Есть журнал, но не наш -- надо бут отдать.)

```

Reported by `mordula` on 2007-01-25 12:16:59

**Comment by gsbelarus** [comment]
```
И что?
```

Reported by `alexandra.gsoftware` on 2007-03-02 14:38:48

**Comment by gsbelarus** [comment]
```
ниче. у нас берется просто неправильно. 
```

Reported by `mordula` on 2007-03-02 15:17:23

**Comment by gsbelarus** [comment]
```
Кто-нибудь еще может сказать, где конкретно и что неправильно, и как оно должно быть,
и где посмотреть???
```

Reported by `alexandra.gsoftware` on 2008-05-16 12:48:06

**Comment by gsbelarus** [comment]
```
Если никто ничего не знает об этом, я поставлю этой ошибке статус invalid в ближайшем
будущем
```

Reported by `alexandra.gsoftware` on 2008-05-22 13:48:42

**Comment by gsbelarus** [comment]
```
По фиксированным ценам расчет НДС идет от суммы скидки, а не от всей стоимости. Более
подробно надо искать первоисточник, я точно знаю что правильный расчет НДС на товары
по фиксированным ценам настроен на Могилевском мясокомбинате и на Березовском
мясокомбинате, настройкой занималась Илона.
```

Reported by `mikleshoihet` on 2008-05-22 14:34:41

**Comment by gsbelarus** [comment]
```
Смотрите Issue 670. Насколько я помню, все исправлялось и работало правильно, если
потом не поломали.
```

Reported by `danilchyk` on 2008-05-26 13:07:29

**Comment by gsbelarus** [comment]
Reported by `danilchyk` on 2013-05-22 12:08:52
- **Status changed**: `Duplicate`
- **Merged into**: #670
